canonical/lxd-ui
# Allow bulk select/delete images in a project and unify pagination placement

## Done

- allow bulk select and delete images
- extract `PageHeader` component
- refactor settings, instance list and project list to use the new header component
- move table pagination always below table

Fixes #618
## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - select one or many images and remove them from a project's image list
    - ensure layout of following pages (also responsive and with panel) is correct:
      - instance list
      - instance detail snapshots
      - profile list
      - image list
      - settings
      - storage volume list
      - custom storage volume snapshot list
      - custom iso list
